documentation/staging/content/userguide/managing-domains/domain-events.md

```diff
@@ -5,20 +5,40 @@ weight = 10
 pre = "<b> </b>"
 +++
 
-#### Contents
+### Contents
 
 - [Overview](#overview)
+- [Overview](#What has been changed in 4.0)
 - [Operator-generated event types](#operator-generated-event-types)
 - [Operator-generated event details](#operator-generated-event-details)
 - [How to access the events](#how-to-access-the-events)
 - [Examples of generated events](#examples-of-generated-events)
 
+### Overview
 
-#### Overview
+This document describes Kubernetes events that the operator generates about resources that it manages, during key points of its processing workflow. These events provide an additional way of monitoring your domain resources. Most of the operator-generated events are associated with a domain resource, and those events are included in the domain resource object as well. Note that the Kubernetes server also generates events for standard Kubernetes resources, such as pods, services, and jobs that the operator generates on behalf of deployed domain custom resources.
 
-This document describes Kubernetes events that the operator generates about resources that it manages, during key points of its processing workflow. These events provide an additional way of monitoring your domain resources. Note that the Kubernetes server also generates events for standard Kubernetes resources, such as pods, services, and jobs that the operator generates on behalf of deployed domain custom resources.
+### What has been changed in 4.0
```

Comment:
What has been changed in 4.0 -> how about "What's New in 4.0" or "What's changed in 4.0"
If you change the heading, don't forget to change the link in the TOC

Reply:
Changed to "What's changed in 4.0". Thanks.

Reply:
Changed to "What's new" as we discussed offline.